.pylintrc

```diff
@@ -11,7 +11,7 @@ profile=no
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS,salt/ext
```

Comment:
I don't think this change was intended to go here.

Reply:
yeah, this one can be removed